src/extract.rs

```diff
@@ -88,30 +79,21 @@ pub struct ExtractedPointLight2d {
 
 pub fn extract_point_lights(
     mut commands: Commands,
-    mut previous_len: Local<usize>,
-    point_lights_query: Extract<Query<(Entity, &PointLight2d, &GlobalTransform, &ViewVisibility)>>,
+    point_lights_query: Extract<Query<(RenderEntity, &PointLight2d, &GlobalTransform, &ViewVisibility)>>,
 ) {
     commands.spawn(ExtractedPointLight2d::default());
```

Comment:
This is also adding an empty entity that should also be removed from `lighting.wgsl:42`.